sunlight-shadow/README.md

```diff
@@ -6,43 +6,28 @@
 * John Samuel
 * Gilles Gesquière
 
-## Sunpath tool
+## Description
 
-You can either run this script from the command line, or import it as a library into your project.
-This tool lets you calculate the sunpath for each hour of an interval of years (ex: 1975-2025), from the point of view of a single point on eath. Described by its latitde and longitude. The result is exported in a csv file, with a format compatible with [sunlight](https://github.com/VCityTeam/Sunlight) and [pySunlight](https://github.com/VCityTeam/pySunlight).
+This project aims to provide tools to calculate sunlight exposure of city objects, using data in the 3DTiles format it is composed of multiple subprojects that all work together to form a coherent toolsuite.
```

Comment:
Precise the input (file formats and required files) for the complete running of this tool (with pysunlight/docker), For example, b3dm and tileset.json